drivers/gles2/rasterizer_canvas_gles2.cpp

```diff
@@ -195,7 +195,6 @@ void RasterizerCanvasGLES2::_draw_polygon(const int *p_indices, int p_index_coun
 		glBufferSubData(GL_ARRAY_BUFFER, buffer_ofs, sizeof(Vector2) * p_vertex_count, p_uvs);
 		glEnableVertexAttribArray(VS::ARRAY_TEX_UV);
 		glVertexAttribPointer(VS::ARRAY_TEX_UV, 2, GL_FLOAT, GL_FALSE, sizeof(Vector2), ((uint8_t *)0) + buffer_ofs);
-		buffer_ofs += sizeof(Vector2) * p_vertex_count;
```

Comment:
While it's true it's not necessary, I assume this was added to ensure that it won't be forgotten if other GL operations were added below the `ARRAY_TEX_UV` one. CC @karroffel

Reply:
Yeah that's the idea. The general pattern is

- set up attribute
- change buffer offset

I did that for each occurance to be consistent. I can't tell if there's something that's still unimplemented there that I just didn't add yet, but I would keep it, it doesn't hurt and expresses intent anyway.